turbo.json

```diff
@@ -15,6 +15,9 @@
     "dev": {
       "cache": false
     },
+    "docs#dev": {
+      "dependsOn": ["schema"]
+    },
```

Comment:
Now that #2706 is merged, we can actually start moving some of these configs into workspaces! (definitely out of scope to do here though!)